lib/index.js

```diff
@@ -121,6 +121,7 @@ class Mcdev {
                 // Other map only with type -> file (for buildDefinitionBulk)
                 const buTypeDelta = {};
                 const typeDelta = {};
+                let deltaCounter = 0;
```

Comment:
not loving increasing logic creep in what is the index.js

Reply:
Open to ideas on how to solve this more efficiently ;-)